src/DumpDocs.php

```
@@ -136,61 +195,129 @@
+                        $descriptors[] = $original;
+                    }
+                }
+            }
+            // If neither id nor href#id is present, skip this descriptor.
+        }
 
-            $original = clone $this->descriptors[$id];
-            $descriptors[] = $original;
+        // Return empty array if no valid descriptors found
+        if (empty($descriptors)) {
+            return [];
```

PHPStan (2.0.3)

Method Koriym\AppStateDiagram\DumpDocs::getInlineDescriptors() should return non-empty-list<Koriym\AppStateDiagram\AbstractDescriptor> but returns array{}.
array{} is empty.

(return.type)